app/repositories/application/OnlineTestRepository.scala

```diff
@@ -46,6 +46,8 @@ trait OnlineTestRepository {
 
```

Comment:
This is a general comment, will be have a separate repository for the different online tests: video interview, etc?

If yes, maybe we should renamed all things related to phase1 to something different maybe OnlineExercise or Phase1OnlineTest?

Anyway I would recommend to split everything in 3.